frontend/src/old-pages/Configure/Components.tsx

```diff
@@ -924,7 +924,19 @@ function HelpTextInput({
   let error = useState([...errorsPath, configKey])
 
   return (
-    <FormField label={name} errorText={error} description={description}>
+    <FormField
+      label={name}
+      errorText={error}
+      description={description}
+      info={
+        <InfoLink
+          ariaLabel={name}
+          helpPanel={
+            <TitleDescriptionHelpPanel title={name} description={help} />
+          }
+        />
+      }
+    >
       <div
         style={{
           display: 'flex',
```

Comment:
suggestion(non-blocking): you could improve this inline style even if you didn't touch it so making it into a const

Reply:
It make sense, I'll get back to it in another PR!